libs/langchain/langchain/retrievers/self_query/base.py

```diff
@@ -12,6 +12,7 @@
     MongoDBAtlasVectorSearch,
     MyScale,
     OpenSearchVectorSearch,
+    PathwayVectorClient,
```

Comment:
this will break backwards compatibility for anyone importing from langchain as harrison mentioned above.